logstash-core/lib/logstash/api/lib/app/commands/system/plugins_command.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+require "app/command"
+
+class LogStash::Api::PluginsCommand < LogStash::Api::Command
+
+  def run
+    { :total => plugins.count, :plugins => plugins }
+  end
+
+  private
+
+  def plugins
+    @plugins ||= find_plugins_gem_specs.map do |spec|
+      { :name => spec.name, :version => spec.version.to_s }
```

Comment:
I believe `to_s` is unnecessary here since converting the hash to json should call this.